README.md

```diff
@@ -53,72 +47,48 @@ So that I can stay constantly tapped in to the shouty box of Chitter
-
-**ActiveRecord**
-- [ActiveRecord ORM](https://guides.rubyonrails.org/active_record_basics.html)
-- [Sinatra, PostgreSQL & ActiveRecord recipe](http://recipes.sinatrarb.com/p/databases/postgresql-activerecord?#article)
-
-
-Notes on functionality:
-------
-
-* You don't have to be logged in to see the peeps.
-* Makers sign up to chitter with their email, password, name and a username (e.g. samm@makersacademy.com, password123, Sam Morgan, sjmog).
-* The username and email are unique.
-* Peeps (posts to chitter) have the name of the maker and their user handle.
-* Your README should indicate the technologies used, and give instructions on how to install and run the tests.
-
-Bonus:
------
-
-If you have time you can implement the following:
-
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
-
-And/Or:
-
-* Work on the CSS to make it look good.
-
-Good luck and let the chitter begin!
-
-Code Review
------------
-
-In code review we'll be hoping to see:
+Installation
+-------
+- $ `git clone git@github.com:CazaBelle/chitter-challenge.git`
+- $ `bundle install`
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+Database Config
+-------
+- $`rake db:create` to dbs that exist in database.yml file)
+- $`rake db:migrate`
+- $`rake db:test:prepare`
+- $`rake db:reset` use if wanting to wipe development table clean
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+QuickStart
+-------
+- $ `rackup`
+- Navigate to `local host:9292` in your browser
 
-Notes on test coverage
-----------------------
+Testing 
+-------
+- $ `rspec`
 
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
+![coverage](images/chitter_coverage.png)
 
-```ruby
-require 'simplecov'
-require 'simplecov-console'
+Demo
+-------
+Homepage
+-------
+![homepage](images/chitter_homepage.png)
+Sign up
+-------
+![signup](images/chitter_signup_page.png)
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
-```
+Log in
+-------
+![signin](images/chitter_login_page.png)
+Profile
+-------
+![profile](images/chitter_profile.png)
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+Extensions
+-------
+* Adding password encryption
+* Styling
+* Ability to tag other users in peeps
+* Start a conversation as by replying to a peep
```

Comment:
Lovely README, I'd have no problem following the instructions to get everything set up locally. Mentioning the technologies used would be a nice addition, especially to highlight your use of Active Record.